cmd/studio.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/utils"
+)
+
+var studioCmd = &cobra.Command{
+	Use:   "studio",
+	Short: "Opens Supabase Studio in your browser",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		url := utils.GetSupabaseDashboardURL()
```

Comment:
This opens the hosted dashboard and not local studio. Is that your intention?

It might be a good idea to support `--local` and `--linked` flag to distinguish these use cases.